packages/api/src/services/lsp-state.ts

```diff
@@ -0,0 +1,127 @@
+import * as uma from "@uma/sdk";
```

Comment:
most of the lsp state fetching logic. heavily uses multicall and is structured similarly to the emp-state service.